frontend/tests/features/wheresMyGene.test.ts

```diff
@@ -592,6 +598,68 @@ describe("Where's My Gene", () => {
     });
   });
 
+  describe("Find Marker Genes", () => {
+    test("Marker gene panel opens and can add genes to dotplot", async ({
+      page,
+    }) => {
+      await goToPage(`${TEST_URL}${ROUTES.WHERE_IS_MY_GENE}`, page);
+
+      await clickDropdownOptionByName({
+        page,
+        selector: getTestID(ADD_TISSUE_ID),
+        name: "lung",
+      });
+
+      await getCellTypeFmgButtonAndClick(page, "muscle cell");
+
+      await getButtonAndClick(page, getTestID(ADD_TO_DOTPLOT_BUTTON_TEST_ID));
+
+      await waitForHeatmapToRender(page);
+
+      const geneLabels = await page.$$(GENE_LABELS_ID);
+      const numGenes = geneLabels.length;
+      expect(numGenes).toBeGreaterThan(0);
+    });
+
+    test("Cell types with no marker genes display warning", async ({
+      page,
+    }) => {
+      await goToPage(`${TEST_URL}${ROUTES.WHERE_IS_MY_GENE}`, page);
+
+      await clickDropdownOptionByName({
+        page,
+        selector: getTestID(ADD_TISSUE_ID),
+        name: "lung",
+      });
+
+      await getCellTypeFmgButtonAndClick(page, "native cell");
+
+      await waitForElement(page, getTestID(NO_MARKER_GENES_WARNING_TEST_ID));
+    });
+
+    test(`Marker scores are always greater than ${FMG_GENE_STRENGTH_THRESHOLD}`, async ({
+      page,
+    }) => {
+      await goToPage(`${TEST_URL}${ROUTES.WHERE_IS_MY_GENE}`, page);
+
+      await clickDropdownOptionByName({
+        page,
+        selector: getTestID(ADD_TISSUE_ID),
+        name: "lung",
+      });
+
+      await getCellTypeFmgButtonAndClick(page, "club cell");
+
+      const markerScores = await page.$$(getTestID(MARKER_SCORES_FMG_TEST_ID));
+
+      for (const markerScore of markerScores) {
+        expect(
+          parseFloat(await markerScore.innerText())
+        ).toBeGreaterThanOrEqual(FMG_GENE_STRENGTH_THRESHOLD);
```

Comment:
should this be `toBeGreaterThan` instead of `ThanOrEqual` ? Since the test description says it's always greater than

Reply:
based on the implementation, it's greater than or equal to. i can update the test description.